inttestutils/buildinfo.go

```diff
@@ -28,3 +35,18 @@ func DeleteBuild(artifactoryUrl, buildName string, artHttpDetails httputils.Http
 		log.Error(string(body))
 	}
 }
+
+func ValidateGeneratedBuildInfo(t *testing.T, buildname, buildNumber string, moduleNames []string, moduleType buildinfo.ModuleType) {
```

Comment:
U think that this function may fail, if it used on a build-info which includes multiple modules, with different modules types. This is because of this assertion:
```
assert.Equal(t, moduleType, module.Type)
```
I  think we should replace this assertion with an addition type check as part of the if.
Also, we may want to change the name of the function from ```ValidateGeneratedBuildInfo``` to ```ValidateGeneratedBuildInfoModule``` to better describe what the function does.

Since the validation can fail due to the type as well, let's add the module type to the the following message:
```
assert.Fail(t, fmt.Sprintf("Module %s is expected to be one of %v", module.Id, moduleNames))
```